interfaces/airVantage/le_avdata.api

```diff
@@ -432,7 +432,7 @@ FUNCTION le_result_t CreateResource
 //--------------------------------------------------------------------------------------------------
 FUNCTION le_result_t SetNamespace
 (
-    Namespace namespace
+    Namespace nameSpace
```

Comment:
I think `Namespace ns` or `Namespace namespaceValue` would be preferable to `Namespace nameSpace` because the camel casing implies two words "name space" rather than the single word "namespace".  Internally, there was some discussion about a more generic solution to this problem.  One idea was to mangle the parameter name in the generated C++ code so that it would be something like "namespaceKw".  This has the advantage that appropriate keyword checks could be added for every new language in the code generator.

Reply:
I like the idea David for the potential workarounding in the code-generator itself, but that could result in weird behavior for languages that can select parameters (Python and possibly Ruby, AFAIK).

I think you can do something such as `myFunction(namespace="a")` for instance. As long as it updates the name only when the name is in a list of keywords that's still better than nothing though, and that would avoid needing to rework each API definition every time we start supporting a new language.

Reply:
I think mangling the parameter name in a predictable way gives the best possible experience for languages where the caller might care about the names of the parameters (eg. keyword arguments).  If that solution is implemented, we need to be careful that mangled name isn't the name of another parameter.